main.nf

```diff
@@ -20,51 +20,47 @@ def helpMessage() {
+      --precursor_mass_tolerance          Mass tolerance of precursor mass (ppm)
+      --fragment_mass_tolerance           Mass tolerance of fragment mass bin (ppm)
+      --fragment_bin_offset               Offset of fragment mass bin (Comet specific parameter)
+      --use_x_ions                        Use x ions for spectral matching in addition
+      --use_z_ions                        Use z ions for spectral matching in addition
+      --use_a_ions                        Use a ions for spectral matching in addition
+      --use_c_ions                        Use c ions for spectral matching in addition
+      --fdr_threshold                     Threshold for FDR filtering
+      --fdr_level                         Level of FDR calculation ('peptide-level-fdrs', 'psm-level-fdrs', 'protein-level-fdrs')
+      --digest_mass_range                 Mass range of peptides considered for matching
+      --activation_method                 Fragmentation method ('ALL', 'CID', 'ECD', 'ETD', 'PQD', 'HCD', 'IRMPD')
+      --enzyme                            Enzymatic cleavage ('unspecific cleavage', 'Trypsin', see OpenMS enzymes)
+      --number_mods                       Maximum number of modifications of PSMs
+      --fixed_mods                        Fixed modifications ('Carbamidomethyl (C)', see OpenMS modifications)
+      --variable_mods                     Variable modifications ('Oxidation (M)', see OpenMS modifications)
+      --num_hits                          Number of reported hits
+      --run_centroidisation               Specify whether mzml data is peak picked or not (true, false)
+      --pick_ms_levels                    The ms level used for peak picking (eg. 1, 2)
+      --prec_charge                       Precursor charge (eg. "2:3")
+      --max_rt_alignment_shift            Maximal retention time shift (sec) resulting from linear alignment      
+      --spectrum_batch_size               Size of Spectrum batch for Comet processing (Decrease/Increase depending on Memory Availability)
+      --description_correct_features      Description of correct features for Percolator (0, 1, 2, 4, 8, see Percolator retention time and calibration) 
+      --klammer                           Retention time features are calculated as in Klammer et al. instead of with Elude.
+      --predict_RT                        Retention time prediction for identified peptides
+      --skip_decoy_generation             Use a fasta databse that already includes decoy sequences
+      --quantification_fdr                Assess and assign ids matched between runs with an additional quantification FDR
+      --quantification_min_prob           Specify a minimum probability cut off for quantification
     Binding Predictions:	
         --predict_class_1                 Whether a class 1 affinity prediction using MHCFlurry should be run on the results - check if alleles are supported (true, false)	
```

Comment:
Indents are off here.

Reply:
Need to add spaces between sections too.

Reply:
https://github.com/Zethson/MHCquant/commit/ea5eb696f5b4db358c039e1a1feed0bcee052b63

Reply:
https://github.com/Zethson/MHCquant/commit/3fea36ec6fe378a9b1b740e4ea2bfbec987663e1